htdocs/accountancy/class/lettering.class.php

```diff
@@ -157,7 +157,7 @@ public function letteringThirdparty($socid)
 					$sql .= " INNER JOIN " . MAIN_DB_PREFIX . "accounting_bookkeeping as bk ON (bk.fk_doc = pay.fk_bank AND bk.code_journal='" . $obj->code_journal . "')";
 					$sql .= " WHERE payfac.fk_paiement = '" . $obj->url_id . "' ";
 					$sql .= " AND bk.code_journal IN (SELECT code FROM " . MAIN_DB_PREFIX . "accounting_journal WHERE nature=4 AND entity=".$conf->entity.") ";
-					$sql .= " AND fac.entity = ".$conf->entity;
+					$sql .= " AND fac.entity IN (".getEntity('invoice',0).")";// We don't share object for accountancy
```

Comment:
We are here in the module accountancy.
The module accountancy must not be managed like other. Everything must always be done on 1 company even if invoice are shared. Because if an invoice is counted into a company, it must never be counted into another. For this reason invoices are never shared into module accountancy, and for this reason we must keep 
AND fac.entity = ".$conf->entity;